### ISSUE TITLE
Private packages are reported as UNLICENSED but check fail with false positive

### ISSUE THREAD

**Issue by otbe** [davglass/license-checker#163]
Hi,

I tried to setup a license check for one of our SPAs. This project is private and gets reported as UNLICENSED when running `license-checker` (btw I would prefer to not add the package to the license report at all). If I run `license-checker --production --onlyAllow 'Apache;BSD;MIT;CDDL;LGPL;ISC'` I get a false positive. It reports a custom license which is a result of parsing the README.md from the SPA itself. The output looks like
```
  "my-spa@0.1.0": {
    "licenses": "UNLICENSED",
    "private": true,
    "publisher": "Me",
    "email": "me@example.com",
    "path": "/my-spa",
    "licenseFile": "/my-spa/README.md"
  },
```

Is there any way to workaround this?

**Comment by jayvdb** [comment]
Also facing this problem.

This was a feature https://github.com/davglass/license-checker/pull/108 by @dancrumb so maybe he can help find the best way to support his needs and ours.

**Comment by feliciafaye** [comment]
I'm also getting a false positive. Did you find any workaround @otbe ?

**Comment by jarrodconnolly** [comment]
Maybe an option for excluding specific modules by name? That may be useful for other case as well as this one.

Then op could run for example:
`license-checker --production --onlyAllow 'Apache;BSD;MIT;CDDL;LGPL;ISC' --excludeModule my-spa`

Just thinking as I am also running into this issue.

**Comment by bmuenzenmeyer** [comment]
I pulled down `license-checker` and altered `lib/license.js`

``` diff
...
var IS_FILE_REFERENCE = /SEE LICENSE IN (.*)/i;
+var UNLICENSED = /UNLICENSED/i;


module.exports = function(str) {
...
```

``` diff
    } else if(PUBLIC_DOMAIN.test(str))  {
        return 'Public Domain';
+    } else if (UNLICENSED.test(str)) {
+        return 'UNLICENSED';
    } else {
```

Creating summary output that looks like this:

```
├─ MIT: 1250
├─ ISC: 107
├─ BSD-3-Clause: 35
├─ Apache-2.0: 30
├─ BSD-2-Clause: 25
├─ BSD*: 14
├─ MIT*: 8
├─ (MIT OR Apache-2.0): 2
├─ UNLICENSED: 2   <--------- yay
├─ (MIT AND Zlib): 1
├─ (WTFPL OR MIT): 1
├─ (GPL-2.0 OR MIT): 1
├─ (MIT AND BSD-3-Clause): 1
└─ (Apache-2.0 OR MPL-1.1): 1
```

This passed all existing tests so far, but likely has ramifications for how license-checker should report compliance according to different flags. This served my use case, but cannot guarantee it would work for everyone.

**Comment by dcousens** [comment]
I understand I can work around this with `excludePackages`,  but, if I have many private packages... this is a mess.

Why is the license not respected if it is `private`?
